i18n/validate.py

```diff
@@ -26,8 +26,12 @@ def validate_po_files(root, report_empty=False):
     for dirpath, __, filenames in os.walk(root):
         for name in filenames:
             __, ext = os.path.splitext(name)
-            if ext.lower() == '.po':
-                filename = os.path.join(dirpath, name)
+            filename = os.path.join(dirpath, name)
+
+            # validate only .po files that are not product of a merge (see generate.py)
+            # if django-partial.po has a problem, then django.po will also, so don't report it.
+            if (ext.lower() == '.po' and
+                    os.path.basename(filename) not in config.CONFIGURATION.generate_merge.keys()):
```

Comment:
Nitpick: this condition is quite long, could you make it shorter and more readable, eg. by moving `config.CONFIGURATION.generate_merge.keys()` to a local variable?